### PR TITLE
HW1: 1, 2, 3

### PR DESCRIPTION
**Pull request by sn2865**

### CODE REVIEW

File: contribution.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+## Project Contributions
```
Comment (timkpaine) [inline contribution.md:1]: rename to `projects/python/aither.md` or whatever language it is in. Also provide a link.

**Comment by sn2865** [comment]
I update tkp2108.md but I should instead update my file

File: project.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
```
Comment (timkpaine) [inline project.md:1]: move to `projects/python/<your project name>.md`, or whatever language it is in

File: sn2865.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+UNI: sn2865
+
+Github: [link](https://github.com/sn2865)
+
+GitHub README: [link](https://github.com/sn2865/sn2865/blob/main/README.md)
+
+## Project Proposal
+
+Proposal: [link](https://github.com/sn2865/project-proposals-s2023/blob/main/project.md)
```
Comment (timkpaine) [inline sn2865.md:13]: use relative link like `./projects/python/aither.md`

File: sn2865.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+## Project Contributions
+
+Contribution: [link](https://github.com/sn2865/project-proposals-s2023/blob/main/contribution.md)
```
Comment (timkpaine) [inline sn2865.md:17]: use relative link

File: tkp2108.md [hunk 5]
```diff
@@ -1,14 +0,0 @@
-# Open Source Development
```
Comment (timkpaine) [inline tkp2108.md:1]: restore this file

**Review by timkpaine — CHANGES_REQUESTED** [review]

**Pull request closed without merging**